File: spec/code_samples/PHP/candidates/get.php
<?php
use Minter\MinterAPI;
use GuzzleHttp\Exception\RequestException;

$api = new MinterAPI('https://minter-node-1.testnet.minter.network:8841');

try {
    $height = 1; // optional, should be positive integer or null
    $response = $api->getCandidates($height);
    print_r($response);
} catch(RequestException $exception) {
    // handle error
}

File: spec/code_samples/Swift/candidates/get.swift
import MinterCore

MinterCoreSDK.initialize(urlString: "https://minter-node-1.testnet.minter.network:8841/")

let manager = CandidateManager.default

manager.candidates(height: "1", completion: { (response, error) in
	print(response)
	print(error)
})
